templates/terraform/resource.erb

```diff
@@ -232,12 +256,21 @@ func resource<%= resource_name -%>Update(d *schema.ResourceData, meta interface{
 	<% end -%>
 	}
 
+  <% if object.custom_code.encoder || object.custom_code.update_encoder -%>
```

Comment:
This block can be simplified using if/elseif

Reply:
Ha, of course.